qmcpy/discrete_distribution/_discrete_distribution.py

```diff
@@ -52,6 +52,39 @@ def pdf(self, x):
         """ ABSTRACT METHOD to evaluate pdf of distribution the samples mimic at locations of x. """
         raise MethodImplementationError(self, 'pdf')
 
+    def plot(self, n, d_vertical=0, d_horizontial =1):
+        """
+        ABSTRACT METHOD for ploting a distrubution, you will also need to import matplotlib.pyplot as plt
+
+        Args:
+            n (int): n is the number of samples that will be calculated 
+
+            d_vertical (int): d_vertical is the index of points that will be plotted on the vertical axis. See below
+            
+
+            d_horizontial (int): d_horizontial is the index of points that will be plotted as the horizontial axis. See below
+
+        d_horizontial and d_vertical explanation:
+            Both of theses numbers act as an index in an array of dimensions for the given object. 
+            The chosen number for d_horizontial or d_vertical will index a given dimension. With the d_horizontial the dimension will be on the 
+            x-axis and the d_vertical index will then become the y-axis. 
+
+            
+
+        The plot method should be defined similar to :
+            fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(16, 6))
+            samples = self.gen_samples(n)
+            ax.scatter(samples[:, d_vertical], samples[:, d_horizontial])
+            plt.show()
+
+        
+        This method does not return anything it just plots a scatter plot.
```

Comment:
return the `fig,ax` in case the user wants to further modify our baseline plots.